src/webgl/program.js

```diff
@@ -94,39 +95,33 @@ export default class Program extends Resource {
       this.gl.useProgram(this.handle);
 
       if (transformFeedback) {
-        if (parameters[GL.RASTERIZER_DISCARD]) {
```

Comment:
Maybe check if `undefined`, and in that case set it to true, otherwise as the user specified it.

If we don't use `withParameters`, we get the risk of an exception leaving the flag set/unset.

Reply:
Since we are now calling `withParameters` inside, program.draw, any users specified setting will be taken care properly without polluting any previous state. As  commented above, not enforcing `rasterizer_discard` during TF flow, gives required flexibility in the API.